### ISSUE TITLE
Color fluid animation

### ISSUE THREAD

**Issue by sherloach** [haren724/open-wallpaper-engine-mac#24]
I really like this fluid wallpaper from wallpaper engine. Does it work with this?
https://steamcommunity.com/sharedfiles/filedetails/?id=1748506393)](https://steamcommunity.com/sharedfiles/filedetails/?id=1748506393)

**Comment by haren724** [comment]
Yes, It definitely works! Tested at v0.5.2 in TestFlight release

Here is the preview:

<img width="1512" alt="Screenshot 2023-09-09 at 03 34 26" src="https://github.com/haren724/open-wallpaper-engine-mac/assets/61335850/02fc5ae7-4048-4d4b-a090-1fa6301dcac7">

**Comment by haren724** [comment]
Due to a Cocoa API limitation (or just because I'm noob to that framework), you can only interact with the wallpaper with mouse clicking and dragging. Slightly different from Wallpaper Engine.

**Comment by sherloach** [comment]
Let me try, thanks.

btw, is there a way to use Wallpaper Engine on Mac?

**Comment by sherloach** [comment]
There are some problems:

- it's seem that the folder of Fluid wallpaper has no mp4 file so your app set the default wallpaper (the blue background in the menu bar)

<img width="2056" alt="Screenshot 2023-09-09 at 10 19 12 am" src="https://github.com/haren724/open-wallpaper-engine-mac/assets/52942350/b6cfa998-8db6-49e4-8871-ce32e9a0b48a">

- I can't use it for multiple desktops, this is my desktop 2, it still show the default wallpaper rather than the Fluid wallpaper

<img width="2056" alt="Screenshot 2023-09-09 at 10 19 33 am" src="https://github.com/haren724/open-wallpaper-engine-mac/assets/52942350/77425e82-8628-422e-91d2-b7336f8d957c">

**Comment by haren724** [comment]
> Let me try, thanks.
> 
> btw, is there a way to use Wallpaper Engine on Mac?

> There are some problems:
> 
> * it's seem that the folder of Fluid wallpaper has no mp4 file so your app set the default wallpaper (the blue background in the menu bar)
> 
> <img alt="Screenshot 2023-09-09 at 10 19 12 am" width="2000" src="https://user-images.githubusercontent.com/52942350/266752757-b6cfa998-8db6-49e4-8871-ce32e9a0b48a.png">
> * I can't use it for multiple desktops, this is my desktop 2, it still show the default wallpaper rather than the Fluid wallpaper
> 
> <img alt="Screenshot 2023-09-09 at 10 19 33 am" width="2000" src="https://user-images.githubusercontent.com/52942350/266752855-77425e82-8628-422e-91d2-b7336f8d957c.png">

ym the Wallpaper Engine on Steam? I think it's impossible

Which version are you currently using? I suggest you could get the latest build from [here](https://testflight.apple.com/join/9fDQc6dA)

**Comment by sherloach** [comment]
oh I just downloaded the latest version, and the first problem is fixed. But it seems like just the first desktop works, the other desktops does not have animation when I click or drag, just a static wallpaper

**Comment by haren724** [comment]
> oh I just downloaded the latest version, and the first problem was fixed. But it seems like just the first desktop works, the other desktops don't have animation when I click or drag, just a static wallpaper

Indeed, That's because I only designed Wallpaper Window which reveals at desktop layer to be initialized once.
Wallpapers on the other desktops is truly static wallpaper, that I'd like to make your system menu bar's tint color to pair with the dynamic wallpaper's.
Multi-display function is on its way.
